src/router.ts

```diff
@@ -0,0 +1,8 @@
+import  traineeRouter  from './ controllers/trainee/index';
+import { Router } from 'express';
+
+const mainRouter = Router();
+mainRouter.use('/trainee', traineeRouter);
+
+export default mainRouter;
```

Comment:
rename this as per the name of the file "router"

Reply:
Done